pkg/parsers/operatingsystem/operatingsystem_linux.go

```diff
@@ -14,19 +14,28 @@ var (
 
 	// file to check to determine Operating System
 	etcOsRelease = "/etc/os-release"
+
+	// used by stateless systems like Clear Linux
+	altOsRelease = "/usr/lib/os-release"
```

Comment:
I'm saying you can use a `[]string` here for both as @calavera said
in tests you just have to backup and restore the string slice instead of both files